requirements.txt

```diff
@@ -15,3 +15,4 @@ rasterio==1.0.7
 protobuf==3.6.*
 scipy==1.1.*
 supermercado==0.0.*
+albumentations==0.3.3
```

Comment:
This should be moved to the pytorch section in the `extra_requirements.json` file since it only applies to those backends.